contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--2.1.0--2.2.0.sql

```diff
@@ -2833,6 +2833,211 @@ GRANT SELECT ON sys.sysobjects TO PUBLIC;
 
 CALL sys.babelfish_drop_deprecated_view('sys', 'sysobjects_deprecated_2_1_0');
 
+ALTER VIEW sys.procedures RENAME TO procedures_deprecated_2_1_0;
```

Comment:
should be `_deprecated_2_2_0` because it will have been deprecated since v2.2.

Reply:
I can do that, but wanted to clear any misunderstanding I have. I thought the naming schema of `_2_1_0` implied this deprecated object was part of version 2.1. So in case we see this deprecated object show up as a dependency somewhere else, we'll know this deprecated object's implementation was from version 2.1 release.

But it looks like we're saying deprecated objects names should indicate which version they became obsolete? Am I correct here? I'm following naming convention in the rest of the upgrade script, which is why I named it with `_2_1_0` suffix.

Reply:
Deprecated object's implementation can be from any previous version, e.g., 2.0 or 1.3. Seems like people are mis-using this. Would `deprecated_in_2_2_0` be more clear?

Reply:
Sure, that makes it more clear. Done, renamed to `procedures_deprecated_in_2_2_0`. I'll be vigilant about me and others doing this naming scheme now.